tests/unit/butterfree/load/writers/test_historical_feature_store_writer.py

```diff
@@ -16,13 +16,16 @@ def test_write(
         historical_feature_set_dataframe,
         mocker,
         feature_set,
+        spark_session,
     ):
         # given
         spark_client = SparkClient()
         spark_client.write_dataframe = mocker.stub("write_dataframe")
+        spark_client.add_table_partitions = mocker.stub("add_table_partitions")
```

Comment:
What would be the need to verify the partition in a unit test of writing? It is tested in the integration tests.

Reply:
I think you don't need to actually check if the partitions were created in the table. But since as you already mocked the method, I think it's important to assert the call, and check if the right partition values were passed to this method while executing the `write` method.